store/sqlstore/upgrade.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 const (
-	CURRENT_SCHEMA_VERSION   = VERSION_5_29_0
+	CURRENT_SCHEMA_VERSION   = VERSION_5_30_0
```

Comment:
For cloud, we're going to set this as the next DB schema until we close the version and move to the next one. We're going to leverage the use of this checker to include or re-execute the migrations for 5.30.

Why? This way we keep the migrations reapply for every rollout/restart we made in the pods to the minimum